conu/apidefs/image.py

```diff
@@ -8,6 +8,69 @@ class Image(object):
     A class which represents an arbitrary container image. It contains utility methods
     to manipulate it.
     """
+    def __init__(self, image_reference, tag=None):
```

Comment:
Assuming "latest" is implied (as the docstring below says), why not use `tag="latest"`?

Reply:
+1

Reply:
changing my mind: I fear that `latest` is docker's thing and other container runtimes might do this differently (I'm not even sure if other runtimes have tags)

Reply:
That's a good point. I guess having `None` as general default is sensible.